src/test/scala/com/actian/spark_vector/vector/VectorConnectionPropertiesTest.scala

```diff
@@ -20,44 +20,69 @@ import org.scalatest.prop.PropertyChecks
 
 class VectorConnectionPropertiesTest extends FunSuite with Matchers with PropertyChecks {
   val validCombos = Table(
-    ("host", "instance", "database", "user", "password", "expectedURL"),
-    ("host.com", "VH", "db", Some("user"), Some("pw"), "jdbc:ingres://host.com:VH7/db"),
-    ("some.host.com", "VW", "db", None, None, "jdbc:ingres://some.host.com:VW7/db"),
-    ("justhost", "99", "mydatabase", None, None, "jdbc:ingres://justhost:997/mydatabase"))
+    ("host",          "instance", "instanceOffset", "port",       "database",   "user",      "password",  "expectedURL"),
+    ("host.com",      Some("VH"), None,             None,         "db",         Some("user"), Some("pw"), "jdbc:ingres://host.com:VH7/db"),
```

Comment:
If instance is really equivalent to installation code, then we should add something like "X1" here as well.